agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4866,6 +4867,23 @@ bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMe
 
     //check if the Request Mode bit is set
     if (request_mode) {
+        //TODO Handle 0 or more then 1 sta in list, currenlty cli steers only 1 client
+        auto request_out = message_com::create_vs_message<
+            beerocks_message::cACTION_APMANAGER_CLIENT_BSS_STEER_REQUEST>(cmdu_tx, mid);
+        if (request_out == nullptr) {
+            LOG(ERROR) << "Failed building ACTION_APMANAGER_CLIENT_BSS_STEER_REQUEST message!";
+            return false;
+        }
+        auto bssid_list = steering_request_tlv->target_bssid_list(0);
+
+        request_out->params().mac            = std::get<1>(steering_request_tlv->sta_list(0));
+        request_out->params().disassoc_timer = steering_request_tlv->btm_disassociation_timer();
+        request_out->params().bssid.mac      = std::get<1>(bssid_list).target_bssid;
+        request_out->params().bssid.channel  = std::get<1>(bssid_list).target_bss_channel_number;
+        request_out->params().disassoc_imminent =
+            steering_request_tlv->request_flags().btm_disassociation_imminent_bit;
+
+        message_com::send_cmdu(ap_manager_socket, cmdu_tx);
```

Comment:
The standard allows the tlv to contain multiple stations and target bssids. It is ok that the current implementation fits to one station and target bssid, but it needs to be documented.
Please a TODO comment that explains that it is a temporary implementation, and what has to be done in the future.

Reply:
There is such comment above:
//TODO Handle 0 or more then 1 sta in list, currenlty cli steers only 1 client